icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/PersonNameFormatterTest.java

```diff
@@ -186,8 +186,10 @@ public void TestPrefixCore() {
                 { "en_US", "LONG",   "MONOGRAM",   "INFORMAL", "DEFAULT", "", "WV" },
 
                 // but Dutch monogram generation _does_ handle the prefix specially
-                { "nl_NL", "LONG",   "MONOGRAM",   "FORMAL",   "DEFAULT", "", "WvP" },
-                { "nl_NL", "LONG",   "MONOGRAM",   "INFORMAL", "DEFAULT", "", "WvP" },
+//                { "nl_NL", "LONG",   "MONOGRAM",   "FORMAL",   "DEFAULT", "", "WvP" },   Temporary change the test because data was set to provisional
+//                { "nl_NL", "LONG",   "MONOGRAM",   "INFORMAL", "DEFAULT", "", "WvP" },
+                { "nl_NL", "LONG",   "MONOGRAM",   "FORMAL",   "DEFAULT", "", "WV" },
+                { "nl_NL", "LONG",   "MONOGRAM",   "INFORMAL", "DEFAULT", "", "WV" },
```

Comment:
Sounds like I have some cleanup and sync to do here too...

Reply:
NO, this is because we reset a lot of personName data to provisional before the beginning of data collection for CLDR 44; provisional data dopes not get picked up for ICU. Once we have the CLDR 44 Survey Tool data in xml and redo the integration, these problems should go away.

Reply:
Oh, okay.